pdc/apps/auth/views.py

```diff
@@ -224,15 +223,18 @@ def get_api_perms(request):
     Return all API perms for @groups and users.
     Format: {resource: {create/read/update/delete: [users, @groups]}}
     """
-
     perms = {}
     ret = get_url_with_resource(request)
 
     for obj in models.ResourcePermission.objects.all():
         name = URL_ARG_RE.sub(r'{\1}', obj.resource.name)
         url = ret[name]
         members = get_users_and_groups(obj)
-        perms.setdefault(name, OrderedDict()).setdefault(obj.permission.name, set()).update(members)
+        groups = sorted(["@" + group_name for group_name in members['groups']])
```

Comment:
do we need to sort it again as we alread sort it in members['groups'](line 154)?